docs/api/covidcast-signals/fb-survey.md

```diff
@@ -307,6 +314,27 @@ also available. These have names beginning `smoothed_w`, such as
 `smoothed_wtested_14d`.
 
 
+## Mental Health Indicators
+
+| Signal | Description | Survey Item |
+| --- | --- | --- |
+| `smoothed_anxious_5d` | Estimated percentage of respondents who reported feeling "nervous, anxious, or on edge" for most or all of the past 5 days | C8 |
+| `smoothed_depressed_5d` | Estimated percentage of respondents who reported feeling depressed for most or all of the past 5 days | C8 |
+| `smoothed_felt_isolated_5d` | Estimated percentage of respondents who reported feeling "isolated from others" for most or all of the past 5 days | C8 |
+| `smoothed_worried_become_ill` | Estimated percentage of respondents who reported feeling very or somewhat worried that "you or someone in your immediate family might become seriously ill from COVID-19" | C9 |
+| `smoothed_worried_finances` | Estimated percentage of respondents who report being very or somewhat worried about their "household's finances for the next month" | C15 |
+
+Some of these questions were present from the earliest waves of the survey, but
+only since Wave 4 have respondents consented to aggregate data being used to
+study other impacts of COVID, such as mental health. These aggregates hence only
+include respondents to Wave 4 and later waves, beginning September 8, 2020.
+
+Weighted versions of these signals, using the [survey weighting described
+below](#survey-weighting) to be more representative of state demographics, are
```

Comment:
```suggestion
below](#survey-weighting) for more representation of state demographics, are
```

Reply:
I think "be more representative" is how you'd typically see this phrased in survey/stats research; a sample is representative of the population if it includes people from a broad range of demographics in the population, and isn't biased towards any particular group